### PR TITLE
Issues - Renaming

### PR DESCRIPTION
**Pull request by orhanuysal-google**

### CODE REVIEW

File: src/linter.cc [hunk 1]
```diff
@@ -204,7 +216,8 @@ LinterResult CheckAliasKeyword(absl::string_view sql,
            if (node->node_kind() == AST_ALIAS) {
              int position =
                  node->GetParseLocationRange().start().GetByteOffset();
-             if (sql[position] != 'A' || sql[position + 1] != 'S') {
+             std::string name = ConvertToUppercase(GetNodeString(node, sql));
```
Comment (a-litvinov) [inline src/linter.cc]: const

File: src/linter.cc [hunk 2]
```diff
@@ -204,7 +216,8 @@ LinterResult CheckAliasKeyword(absl::string_view sql,
...
+             if (name.substr(0, 2) != "AS") {
```
Comment (a-litvinov) [inline src/linter.cc]: absl::StartsWith

File: src/runner.cc [hunk 3]
```diff
@@ -114,7 +114,7 @@ void run(std::vector<std::string> sql_files, Config config) {
       continue;
     }
     std::string str = ReadFile(filename);
-    // PrintASTTree(str);
+    PrintASTTree(str);
```
Comment (a-litvinov) [inline src/runner.cc:117]: Is this on purpose?

File: src/runner.cc [hunk 4]
```diff
@@ -111,6 +114,7 @@ void run(std::vector<std::string> sql_files, Config config) {
       continue;
     }
     std::string str = ReadFile(filename);
+    // PrintASTTree(str);
```
[the remark this answers is not in the document]
Comment (orhanuysal-google) [reply src/runner.cc]: fixed in later PR

File: docs/config.md [hunk 5]
```diff
@@ -15,5 +15,6 @@ nolint: ["parser-failed", "alias"];
 |string|end_line|'\n'|End line character used to separate lines|
 |int32|line_limit|100|Maximum number of characters one line should contain|
 |string|allowed_indent|' '(whitespace)|Allowed indentation character(usually ' ' or '\t')|
-|bool|single_quote|true|Whether single or double quote will used in sql file, checked by this [rule](config.md#single-or-double-quote)|
+|bool|single_quote|true|Whether single or double quote will used in sql file, checked by this [rule](checks.md#single-or-double-quote)|
```
[the remark this answers is not in the document]
Comment (orhanuysal-google) [reply docs/config.md:18]: Changed

File: src/linter.cc [hunk 6]
```diff
@@ -131,17 +131,29 @@ LinterResult CheckUppercaseKeywords(absl::string_view sql,
 
   if (!status.ok()) return LinterResult(status);
 
+  // Some special words in tokenizer is defined as "Keyword",
```
[the remark this answers is not in the document]
Comment (orhanuysal-google) [reply src/linter.cc]: Changed

File: src/linter.cc [hunk 7]
```diff
@@ -131,17 +131,29 @@ LinterResult CheckUppercaseKeywords(absl::string_view sql,
...
+  // but linter shouldn't complain for them. The list can change easily, and
+  // every word in the list should be uppercase.
+  std::vector<std::string> nolint_words{"TIME", "DATE", "TRUE", "FALSE"};
```
[the remark this answers is not in the document]
Comment (orhanuysal-google) [reply src/linter.cc]: fixed in later PR

File: src/linter.cc (hunk 1 above)
Comment (orhanuysal-google) [reply src/linter.cc]: Changed

File: src/linter.cc (hunk 2 above)
Comment (orhanuysal-google) [reply src/linter.cc]: Changed

File: src/runner.cc (hunk 3 above)
Comment (orhanuysal-google) [reply src/runner.cc:117]: fixed in later PR, it is for debugging.

**Pull request merged**